markdown/en/ff-asn.md

```diff
@@ -25,6 +25,32 @@ If you set the ``[filter-style]` attribute properly the `ff-asn-group` acts as d
 </ff-asn>
 ```
 
+## Using native select box
+
+In version 1.2.7 we introduced native select-box support for `ff-asn-group`.
+Additionally `<option>` element does supports data binding when using
```

Comment:
Got some grammar quirks here.
What about:
Additionally, the `<option>` element supports data binding......